audit-triplestore-web/src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ jms.fcrepoEndpoint=${fcrepo.jms.endpoint:topic:fedora}
 jms.brokerUrl=failover:(tcp://${fcrepo.jms.host:localhost}:${fcrepo.dynamic.jms.port:61616})?startupMaxReconnectAttempts=1&initialReconnectDelay=1
 triplestore.baseUrl=${fcrepo.audit.triplestore.baseUrl:localhost:3030/test/update}
 event.baseUri=${fcrepo.audit.baseUri:http://example.com/event}
+audit.container=${fcrepo.audit.container:/audit}
```

Comment:
should the default value have a trailing slash? in case there is e.g. an `/auditory` node in the repository